impit-python/test/async_test.py

```diff
@@ -86,7 +86,7 @@ async def test_passing_string_body_in_data(self, browser: Browser) -> None:
 
         response = await impit.post(
             get_httpbin_url('/post'),
-            data=bytearray('{"Impit-Test":"foořžš"}', 'utf-8'),
+            content=bytearray('{"Impit-Test":"foořžš"}', 'utf-8'),
```

Comment:
This is actually the meaning of this test (that the `data` argument accepts binary data). 

Not having this in the public type is intentional (as far as I understand), so our interface matches the interface of `httpx`. See #97 for more details.

Reply:
Undo & type ignore.